project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/action/FunctionMetricsDescroption.java

```diff
@@ -0,0 +1,103 @@
+package com.samsung.sec.dexter.eclipse.ui.action;
+
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.eclipse.jface.action.IAction;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.ui.IObjectActionDelegate;
+import org.eclipse.ui.IViewPart;
+import org.eclipse.ui.IWorkbenchPart;
+
+import com.samsung.sec.dexter.core.config.DexterConfig;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.core.util.DexterClient;
+//import com.samsung.sec.dexter.eclipse.ui.notifier.NotificationType;
+//import com.samsung.sec.dexter.eclipse.ui.notifier.NotifierDialog;
+import com.samsung.sec.dexter.eclipse.ui.util.EclipseUtil;
+import com.samsung.sec.dexter.eclipse.ui.view.AnalysisLog;
+import com.samsung.sec.dexter.eclipse.ui.view.FunctionMetricsView;
+
+public class FunctionMetricsDescroption implements IObjectActionDelegate {
```

Comment:
Descroption -> Description.
And If this is ActionDelegate, please use like the below formula:
Verb + Object + "ActionDelegate"
eg) OpenFunctionMetricsActionDelegate